app/assets/stylesheets/provider/_header.scss

```diff
@@ -5,7 +5,7 @@ $master-ribbon-height: line-height-times(1);
   height: 76px; // HACK: hardcoded value of PF4 Header hight.
 
   &--master {
-    height: calc(76px + $master-ribbon-height); // HACK: hardcoded value of PF4 Header hight.
+    height: calc(76px + #{$master-ribbon-height}); // HACK: hardcoded value of PF4 Header hight.
```

Comment:
so the problem is using px as that won't scale the height

Reply:
that's why we use `height: line-height-times(63 / 20);` in currently deployed css

Reply:
Do you mean we use that instead of the whole `calc`? But it's be better to have the height as a constant.

Reply:
- I don't think calc is needed as that would move the operation to the browser. Just sass arithmetics will do `+`
- Heights are defined in `line-height-times` so let's stick with that.

$ribbon-height: line-height-times(…)
$master-ribbon-height: $ribbon-height + line-height-times(…)

Reply:
OK, header's height can be in line-height-times but the exact PF value is 76px.